spec/controllers/cases_controller/destroy_spec.rb

```diff
@@ -8,22 +8,35 @@
 
     before { sign_in manager }
 
-    it 'authorises' do
-      expect{
+    context 'FOI case' do
+      it 'authorises' do
+        expect{
+          delete :destroy, params: params
+        }.to require_permission(:destroy?)
+                 .with_args(manager, unassigned_case)
+      end
+
+      it 'sets @case' do
         delete :destroy, params: params
-      }.to require_permission(:destroy?)
-               .with_args(manager, unassigned_case)
-    end
+        expect(assigns(:case)).to eq unassigned_case
+      end
 
-    it 'sets @case' do
-      delete :destroy, params: params
-      expect(assigns(:case)).to eq unassigned_case
+      it 'marks the case as deleted' do
+        delete :destroy, params: params
+        unassigned_case.reload
+        expect(unassigned_case.deleted?).to be true
+      end
     end
 
-    it 'marks the case as deleted' do
-      delete :destroy, params: params
-      unassigned_case.reload
-      expect(unassigned_case.deleted?).to be true
+    context 'ICO case' do
```

Comment:
Should there also be a SAR version of this test?

Reply:
As in a normal SAR case or an ICO SAR case?
I thought it about a normal SAR case, but this ticket was about ICOs, I can add one in though.
ICO SAR cases I think are similar enough to ICO FOIs, but equally not too much work to add it in so can do

Reply:
Sorry, I forgot we had ICO SAR, I was referring to standard SAR cases.

Reply:
I'll do both I think, It's so quick might as well!